tests/tool-bazel.yaml

```
@@ -0,0 +1,5 @@
+- desc: bazel cache should be persisted to /workspaces
+  command: [file, /workspace/.bazel]
```

In case my other `suggestion` is considered, this one below would then be necessary too for emulating a workspace-session behavior.
```suggestion
  entrypoint: [env, GITPOD_REPO_ROOT=/workspace, bash, -ci]
  command: [file, /workspace/.bazel]
```